test/e2e/resources/components/componentGlobalUsage.brs

```diff
@@ -0,0 +1,7 @@
+sub main()
+    ' create an object that uses m.global
```

Comment:
Maybe clarify comment, "create an object that writes value to global, create another object that tries to read that value from global".